### PR TITLE
Documentation for new Archipelago space option

### PR DESCRIPTION
**Pull request by KidsSeeGhosts**
Added documentation for configuring new space option, Archipelago. Describes setup on Archipelago side and follows format of describing existing Fields as the other space documentation.

### CODE REVIEW

File: administrators.rst [hunk 1]
```diff
@@ -181,6 +181,46 @@ more information.
...
+* Click Add content type. Name the content type AIP. 
+* Add a single strawberry field only to the content type, and this will complete setup to allow transfers from Archivematica. 
+* Ensure your user has write permissions via JSON API. To do this, on Archipelago under Administration > Configuration > JSON:API, tick the box "Accept all JSON:API create, read, update, and delete operations." to allow JSON API writes to be made to archipelago. Specific permissions for the AIP content type can also be set under Administration > Struct > Content types > AIP > Manage permissions, where you can select if only authenticated users or anonymous users can create AIPs.
+
+For more info on creating the content type in Archipelago, see drupal instructions `here <https://www.drupal.org/docs/user_guide/en/structure-content-type.html>`_.
+
+
+**In Storage Service:**
+AIPs can be sent to an Archipelago instance via JSON API calls. The Archipelago instance URL is required, as well as an Archipelago username and password with write permissions on the platform. 
+* When creating the space, the staging path should be set to the default `/var/archievmatica/storage_service`.
```
Comment (Dhwaniartefact) [inline administrators.rst]:
```suggestion
* When creating the space, the staging path should be set to the default `/var/archivematica/storage_service`. 
```

File: administrators.rst [hunk 2]
```diff
@@ -181,6 +181,46 @@ more information.
...
+
+During the transfer to Archipelago, the AIP file is uploaded first. If file uploads successfully, a new AIP type entity is created in Archipelago. 
+If the AIP file doesn't uploaded successfully, the new AIP entity is not created in Archipelago. The successful entity contains the uploaded file, and the corresponding Dublin Core metadata extracted from the mets.xml file associated with the AIP.
+
+Fields:
+
+* **Size**: the maximum size allowed for this space. Set to 0 or leave blank
+  for unlimited. This field is optional.
+* **Path**: the local path on the Storage Service machine to the CIFS share. This field is optional for Archipelago.
+* **Staging Path**: the absolute path to a staging area. When creating the space, the staging path should be set to the default `/var/archievmatica/storage_service`.
```
Comment (Dhwaniartefact) [inline administrators.rst]:
```suggestion
* **Staging Path**: the absolute path to a staging area. When creating the space, the staging path should be set to the default `/var/archivematica/storage_service`. 
```

File: administrators.rst [hunk 3]
```diff
@@ -181,6 +181,46 @@ more information.
...
+Fields:
+
+* **Size**: the maximum size allowed for this space. Set to 0 or leave blank
+  for unlimited. This field is optional.
+* **Path**: the local path on the Storage Service machine to the CIFS share. This field is optional for Archipelago.
+* **Staging Path**: the absolute path to a staging area. When creating the space, the staging path should be set to the default `/var/archievmatica/storage_service`. 
+* **Archipealgo URL**: the hostname of the Archipelago web instance or IP address with port,
+  e.g. ``archipelago.example.com:8443``
+* **Archipelago username**: Archipelago username with write access required
+* **Archipelago password**: Arcihpelago password with write access required
```
Comment (Dhwaniartefact) [inline administrators.rst]:
```suggestion
* **Archipelago password**: Archipelago password with write access required
```

**Comment by KidsSeeGhosts** [comment]
@Dhwaniartefact pushed to fix 3 typos.

**Review by Dhwaniartefact — APPROVED** [review]
Thank you for the contribution!

**Pull request merged**